### DISCUSSION TITLE
add new ad to regex

### DISCUSSION THREAD

**Discussion by Gen3ric** [KBlixt/subcleaner#10] [General]
I've been getting this ad a lot recently and it doesn't seem to be covered by the regex: "Someone needs to stop Clearway Law.
Public shouldn't leave reviews for lawyers."
Could it possibly be added to it?

**Comment by KBlixt** [comment]
That's a very weird ad 🤔...

Nevertheless, the script should now recognize it as an ad.

Thank you for the tip and helping with improving the regex 👍

**Comment by Gen3ric** [comment]
Yeah, I'm guessing anyone can buy ad space to say whatever in these ads.

Thanks for adding it and for this awesome tool!
